### PR TITLE
sci-libs/dcmtk: version bump 3.6.5 and prefix fix

### PR DESCRIPTION
**Pull request by TheChymera**
Package-Manager: Portage-2.3.78, Repoman-2.3.17
Signed-off-by: Horea Christian <chr@chymera.eu>

### CODE REVIEW

**Comment by TheChymera** [comment]
Solves Prefix issues recently determined: https://github.com/IBT-FMI/SAMRI/issues/102#issuecomment-562818536

**Comment by TheChymera** [comment]
@juippis and this one? :) Thanks for your help!

File: sci-libs/dcmtk/dcmtk-3.6.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	virtual/jpeg:0
+	png? ( media-libs/libpng:* )
+	ssl? ( dev-libs/openssl:0= )
+	tcpd? ( sys-apps/tcp-wrappers )
+	tiff? ( media-libs/tiff:0 )
+	xml? ( dev-libs/libxml2:2 )
+	zlib? ( sys-libs/zlib )"
+DEPEND="${RDEPEND}
+	media-gfx/graphviz
+	doc? ( app-doc/doxygen )"
```
Comment (juippis) [inline sci-libs/dcmtk/dcmtk-3.6.5.ebuild]: I believe these to be BDEPEND with EAPI-7.

File: sci-libs/dcmtk/dcmtk-3.6.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+DESCRIPTION="The DICOM Toolkit"
+HOMEPAGE="https://dicom.offis.de/dcmtk.php.en"
+SRC_URI="http://dicom.offis.de/download/dcmtk/release/${P}.tar.gz"
```
Comment (juippis) [inline sci-libs/dcmtk/dcmtk-3.6.5.ebuild]: `https://` whenever available.

File: sci-libs/dcmtk/dcmtk-3.6.5.ebuild (hunk 1 above)
Comment (TheChymera) [reply sci-libs/dcmtk/dcmtk-3.6.5.ebuild]: certainly not graphviz.

**Comment by TheChymera** [comment]
@juippis what about now?

**Pull request closed without merging**